pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "aerie-cli"
-version = "1.0.2"
+version = "1.1.0"
 description = "aerie-cli"
 authors = ["Gavin Martin <gavin.c.martin@jpl.nasa.gov>", "Mihir Kamble <mihir.m.kamble@jpl.nasa.gov>"]
```

Comment:
Why aren't you listed as an author @cartermak?

Reply:
Old file. Should probably update to just be "JPL" or something and let the commit contributions speak for themselves?

Reply:
That or having the main Points of Contact.